src/loaddb/load_sa_loader.cpp

```diff
@@ -6444,7 +6444,6 @@ ldr_sa_load (load_args *args, int *status, bool *interrupted)
 						 LOADDB_MSG_LAST_COMMITTED_LINE), lastcommit);
 		}
 	      *interrupted = true;
-	      *status = 3;
```

Comment:
The previous code terminates with exit code 3 when an interrupt occurs.
However, in the changed code, if an interrupt occurs and no error occurs, it terminates with exit code 0. The behavior is different from the previous code. What do you think?

Reply:
If interrupted is true or status != 0 after ldr_sa_loader() is called, it goes down, so there doesn't seem to be a problem in that case.

`ldr_sa_load (&args, &status, &interrupted);
 if (interrupted || status != 0)
        {
          // failed
          db_end_session ();
          db_shutdown ();
          goto error_return;
        }
`

Reply:
Thank you.
As you commented, if an interrupt occurs and there are no errors, it may exit with 0.
So I will change my code according to your comments.